packages/api-server/sqlite_local_config.py

```diff
@@ -8,7 +8,7 @@
 config.update(
     {
         "db_url": f"sqlite://{run_dir}/db.sqlite3",
-        "static_directory": f"{run_dir}/static",  # The directory where static files should be stored.
+        "static_directory": "static",  # The directory where static files should be stored.
```

Comment:
This will mix docs bundles with run time files, we should keep them separated. The docs bundle should be part of the python package and the run time path should be configurable.